source/extensions/common/wasm/null/null_plugin.cc

```diff
@@ -478,6 +478,21 @@ Plugin::Context* nullVmGetContext(uint32_t context_id) {
   return static_cast<NullPlugin*>(null_vm->plugin_.get())->getContext(context_id);
 }
 
+Plugin::Context* nullVmGetContext(uint32_t context_id) {
+  auto null_vm = static_cast<NullVm*>(current_context_->wasmVm());
+  return static_cast<NullPlugin*>(null_vm->plugin_.get())->getContext(context_id);
+}
+
+Plugin::ContextBase* nullVmGetContextBase(uint32_t context_id) {
+  auto null_vm = static_cast<NullVm*>(current_context_->wasmVm());
+  return static_cast<NullPlugin*>(null_vm->plugin_.get())->getContext(context_id);
+}
+
+Plugin::ContextBase* nullVmGetContext(uint32_t context_id) {
+  auto null_vm = static_cast<NullVm*>(current_context_->wasmVm());
+  return static_cast<NullPlugin*>(null_vm->plugin_.get())->getContext(context_id);
+}
+
```

Comment:
Too much copy&paste?

Reply:
Sorry, still getting my workflow back together on the new VM.